### PR TITLE
Default server task to use production config and turn off DEBUG

### PR DESCRIPTION
**Pull request by jcox-dev**
# Motivation and Context

A few quick fixes to turn off debug when using ProductionConfig.

# How to test?

`make serve`  should spin up four gunicorn workers 

NB: will require .env (try: [local.env](https://github.com/ONSdigital/respondent-home-ui/blob/521031cc292df32681dcb2282ae703dd916de26f/tests/test_data/local.env) from the integration branch).

### CODE REVIEW

**Comment by emilioward** [comment]
Tried to run but had some errors will speak to @jcox-dev about it

File: tests/wait_for_services.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def __str__(self) -> str:
+        return f'Healthcheck fails for {self.service.lower()}'
+
+
+def retry_if_http_error(exception):
+    print(f'error has occurred: {str(exception)}')
+    return isinstance(exception, RequestException) or isinstance(exception, HealthCheckException)
+
+
+@retry(retry_on_exception=retry_if_http_error, wait_fixed=10000, stop_max_delay=600000, wrap_exception=True)
```
Comment (damien-lloyd) [inline tests/wait_for_services.py]: Shouldn't we also add a maximum number of retries before ceasing?

Comment (jcox-dev) [reply tests/wait_for_services.py]: @damien-lloyd `stop_max_delay=600000`.

**Comment by jcox-dev** [comment]
@AdamHawtin oops. Forgot to push up the local.env file. Losing track with multiple PR's in flight, clearly!

**Comment by jcox-dev** [comment]
NB: Newline literals (`\n`) in JSON strings cause the decoder to throw a wobbly for some environments.

**Pull request merged**